spec/controllers/spree/checkout_controller_spec.rb

```diff
@@ -84,6 +92,18 @@
           post :update, params: { state: 'confirm' }
           expect(response).to redirect_to spree.order_path(order)
         end
+
+        context 'and not deafult locale' do
+          it 'redirects to the tokenized order view with a non default locale' do
+            if Spree.version.to_f > 3.6
+              request.cookie_jar.signed[:token] = 'ABC'
+            else
+              request.cookie_jar.signed[:guest_token] = 'ABC'
```

Comment:
I would assume here that version is above 3.6, you can always write a comment in the test saying something like "spree version higher than 3.6 required"